### PR TITLE
Event cache test

### PR DESCRIPTION
**Pull request by zbud-msft**
### Description of PR

Summary:

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205
- [ ] 202305

#### What is the motivation for this PR?

Add testcase for event cache testing

#### How did you do it?

Publish events via publish tool, then listen for events and send additional events. Should receive all N events and should see so in stats.

#### How did you verify/test it?

Manual testing/pipeline

### CODE REVIEW

File: tests/telemetry/events/event_utils.py [hunk 1]
```diff
@@ -45,3 +48,29 @@ def restart_monit(duthost):
...
+    with open(ip_file, "w") as f:
+        for i in range(start_idx, end_idx + 1):
+            json_string = f'{{"test-event-source:test": {{"test_key": "test_val_{i}"}}}}'
+            f.write(json_string + '\n')
+    dest = "~/" + json_file
+    duthost.copy(src=ip_file, dest=dest)
+
+
+def event_publish_tool(duthost, json_file):
+    ret = duthost.shell("python ~/events_publish_tool.py -f ~/{}".format(json_file))
```
Comment (qiluo-msft) [inline tests/telemetry/events/event_utils.py:64]: >events_publish_tool [](http://example.com/codeflow?start=34&length=19)

Where does this py file coming from?

**Comment by qiluo-msft** [comment]
Do not mixing bug fix / test improvement / new testcase in one PR. Let's focus on only one aspect in one PR.

File: tests/telemetry/events/event_utils.py (hunk 1 above)
Comment (zbud-msft) [reply tests/telemetry/events/event_utils.py:66]: Part of eventd in image: https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-eventd/tools/events_publish_tool.py

**Comment by zbud-msft** [comment]
> Do not mixing bug fix / test improvement / new testcase in one PR. Let's focus on only one aspect in one PR.

Done

File: tests/telemetry/events/event_utils.py (hunk 1 above)
Comment (qiluo-msft) [reply tests/telemetry/events/event_utils.py]: I see the py file copied from eventd container to the host. Is it possible keep it in original eventd container, and just `docker exec` ?

Comment (zbud-msft) [reply tests/telemetry/events/event_utils.py]: Yes, no longer copy py file from eventd to host, but instead utilize docker exec

**Review by qiluo-msft — DISMISSED** [review]

**Review by qiluo-msft — APPROVED** [review]

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**